packages/@okta/vuepress-theme-prose/assets/css/abstracts/_colors.scss

```diff
@@ -1,117 +1,255 @@
+  --c-yellow-base: #ebd551;
+  --c-yellow-dark: #bc960f;
+
+  --c-warning-lightest: #feefe2;
+  --c-warning-base: #f47c25;
+  --c-warning-dark: #cb5500;
+
+  --c-danger-lightest: #fee6ea;
+  --c-danger-base: #dd0744;
+  --c-danger-dark: #b80047;
+
+  --c-cyan-lightest: #c6fafa;
+  --c-cyan-base: #007f9c;
+  --c-cyan-dark: #004d6b;
+  
+  --c-violet-lightest: #f3eafe;
+  --c-violet-base: #8d6e97;
+  --c-violet-dark: #5a13b4;
+
+  --c-magenta-base: #c72c86;
+
+  --c-code-purple: #965ABE;
+  --c-code-orange: #F19B56;
+  --c-code-lightblue: #8BC5EB;
+  --c-code-lightgray: #C4C4C4;
+
+  --c-cerise-red: #E22866;
+
+  --c-link-blue: #00297A;
+  --c-link-lightgray: #B5D0DF;
+
+  --c-body-lightgray: #DCDCDC;
+  --c-body: #ffffff;
+  --c-tile-gray: #8C8C96;
+  --c-form-gray: #C1C1C8;
+
+  // VARIABLES COLORS FOR LIGHT THEME
+  --c-cerise: var(--c-cerise-red);
+
+  --c-body-base: var(--c-lightgray-200);
+  --c-body-lighter: var(--c-lightgray-300);
+  --c-header-base: var(--c-blue-900);
+  
+  --c-text-darker: var(--c-gray-900);
+  --c-text-base: var(--c-gray-700);
+  --c-text-lighter: var(--c-gray-300);
+
+  --c-link-base: var(--c-blue-500);
+  --c-link-darkblue: var(--c-link-blue);
+  --c-link-footer: var(--c-link-lightgray);
+
+  --c-code-base: var(--c-gray-000);
+
+  --c-code-keyword: var(--c-code-purple);
+  --c-code-define: var(--c-code-orange);
+  --c-code-property: var(--c-code-lightblue);
+  --c-code-string: var(--c-code-lightgray);
+
+  --c-body-border: var(--c-body-lightgray);
+  --c-body-white: var(--c-body);
+  --c-tile-angle: var(--c-tile-gray);
+  --c-form-input-border: var(--c-form-gray);
+
+}
+.dark-theme {
+  
+  // PALETTE DARK THEME
+  --c-gray-900: #17171B;
+  --c-gray-800: #1D1D21;
+  --c-gray-700: #232327;
+  --c-gray-600: #41414B;
+  --c-gray-500: #6E6E78;
+  --c-gray-400: #8C8C96;
+  --c-gray-300: #AAAAB4;
+  --c-gray-200: #DCDCDC;
+  --c-gray-100: #EBEBED;
+  --c-gray-000: #F5F5F6;
+
+  --c-lightgray: #212125;
+
+  --c-blue-300: #92BCFF;
+  --c-blue-800: #234F97;
+
+  --c-code-purple: #8D6E97;
+  --c-code-orange: #E97107;
+  --c-code-lightblue: #92BCFF;
+  --c-code-lightgray: #8C8C96;
+
+  // VARIABLES COLORS FOR DARK THEME
+  --c-body-base: var(--c-gray-900);
+  --c-body-lighter: var(--c-gray-800);
+  --c-header-base: var(--c-gray-700);
+  
+  --c-text-darker: var(--c-white);
+  --c-text-base: var(--c-gray-400);
+  --c-text-lighter: var(--c-gray-000);
+
+  --c-link-base: var(--c-blue-300);
+  --c-code-base: var(--c-gray-900);
+
+  --c-lightgray-block: var(--c-lightgray);
+  
+  --c-code-keyword: var(--c-code-purple);
+  --c-code-define: var(--c-code-orange);
+  --c-code-property: var(--c-code-lightblue);
+  --c-code-string: var(--c-code-lightgray);
+
+}
+
 $colors: (
+  'homepage': (
+    'base': var(--c-gray-900),
+  ),
+  'assurance': (
+    'base': var(--c-lightgray-block),
+  ),
+  'customize': (
+    'base': var(--c-lightgray-block),
+  ),
+  'partners': (
+    'base': var(--c-lightgray-block),
+  ),
```

Comment:
are these new colors for backgrounds? maybe add them to body
'body': (
  'homepage': var(--c-gray-900),
  'assurance': var(--c-lightgray-block),
  'partners': var(--c-lightgray-block)...
)